### DISCUSSION TITLE
Building a Web Version of Your Mastodon Archive with Eleventy

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#28] [Comments]
# Building a Web Version of Your Mastodon Archive with Eleventy

Turning your Mastodon archive into a web site with Eleventy

https://www.raymondcamden.com/2024/07/04/building-a-web-version-of-your-mastodon-archive-with-eleventy

**Comment by robertz** [comment]
This is pretty neat.
